README.asciidoc

```diff
@@ -357,7 +357,7 @@ Maven.resolver().loadPomFromClassLoaderResource("/path/to/pom.xml").resolve("G:A
 ----
 
 Resolving artifacts defined in effective POM::
-ShrinkWrap Resolvers allows you to artifacts defined with specific scope into list of artifacts to be resolved. This way, you don't need to alter your tests if you change dependencies of your application. You can either use +importDependencies(ScopeType...)+ or convenience methods, that cover the most frequent usages (+importRuntimeDependencies()+, +importTestDependencies()+ and +importRuntimeAndTestDependencies()+:
```

Comment:
BTW, I realize the wording of this sentence is somewhat wrong, but I was unsure how to fix it.